calcom/cal.com
# fix: [CAL-3374] Says Number is not Verified, but it is Verified

## What does this PR do?
This PR removes the error not yet verified if the number is verified.

Fixes #14468 

Before  making changes
![image](https://github.com/calcom/cal.com/assets/66114276/c97e9cbc-b48c-467c-b383-ae33220056f2)

After making changes

![image](https://github.com/calcom/cal.com/assets/66114276/982bd7e3-7386-4c0c-8035-0df967e9929b)


## Type of change



- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?
Before testing this make sure your twilio credentials are configured in env.

1. Create a workflow of type send sms to specific number.
2. Add the number which is not verified yet.
3. Type a random verification code(not the correct ones.)
4. Click on save workflow.
5. Now you will see not verified yet. 
6. Now type the correct verification code.
7. Now you will see number verified and no not yet verified.
